src/bounce_solution/bounce_solution.cpp

```diff
@@ -754,14 +754,11 @@ double BounceSolution::CalcTempAtFalseVacFraction(const double &false_vac_frac)
     this->SetStoredTemp(T_middle); // update temp storage for inner integral
     double IatT = prefac * Nintegrate_Outer(*this).result;
 
-    while ((std::abs(T_up / T_down - 1) >
-            RelativeTemperatureInCalcTempAtFalseVacFraction *
-                MarginOfCalcTempAtFalseVacFractionBeforeFailure) and
-           (not almost_the_same(
-               int_at_false_vac_frac,
-               IatT,
-               RelativeErrorInCalcTempAtFalseVacFraction *
-                   MarginOfCalcTempAtFalseVacFractionBeforeFailure)))
+    while (not(std::abs(T_up / T_down - 1) <
+                   RelativeTemperatureInCalcTempAtFalseVacFraction and
+               almost_the_same(int_at_false_vac_frac,
```

Comment:
In the unit test the almost_same part of the while condition is never hit.
Is this just a because the choice of test point or is this in general only a fallback check?